src/redux-enhancers/undoRedoReducerEnhancer.ts

```diff
@@ -212,6 +212,20 @@ const undoRedoReducerEnhancer: StoreEnhancer<any> =
         return newState
       }
 
+      // if an undoable action is of type importText, update inversePatches adding latest patch of actions which contains accurate patches for importText action
+      if (actionType === 'importText') {
+        // we do not want to update inversePatches  when importText runs initially for tutorial event
```

Comment:
I recommend adding one more sentence to this comment explaining what will happen if we don't short circuit here. This helps to complete the question of why this code exists, and makes it clear to future developers what depends on this code.

What is a "tutorial event"? I realize that `importText` is executed at some point during the app's initialization, but I don't see the connection to the tutorial. Maybe we can be more specific here.